src/main/kotlin/no/nav/syfo/api/endpoints/OppfolgingsoppgaveApi.kt

```diff
@@ -27,63 +23,113 @@ fun Route.registerOppfolgingsoppgaveApi(
+            call.checkVeilederTilgang(
+                action = API_ACTION,
+                veilederTilgangskontrollClient = veilederTilgangskontrollClient,
+            ) {
+                val uuid = UUID.fromString(call.parameters[huskelappParam])
+                val requestDTO = call.receive<EditedOppfolgingsoppgaveDTO>()
 
-            oppfolgingsoppgaveService.addVersion(
-                existingOppfolgingsoppgaveUuid = uuid,
-                newVersion = requestDTO
-            )
-                ?.let { createdOppfolgingsoppgaveVersjon ->
-                    call.respond(
-                        HttpStatusCode.Created,
-                        OppfolgingsoppgaveResponseDTO.fromOppfolgingsoppgave(createdOppfolgingsoppgaveVersjon)
+                oppfolgingsoppgaveService.addVersion(
+                    existingOppfolgingsoppgaveUuid = uuid,
+                    newVersion = requestDTO
+                )
+                    ?.let { createdOppfolgingsoppgaveVersjon ->
+                        call.respond(
+                            HttpStatusCode.Created,
+                            OppfolgingsoppgaveResponseDTO.fromOppfolgingsoppgave(createdOppfolgingsoppgaveVersjon)
+                        )
+                    }
+                    ?: call.respond(
+                        HttpStatusCode.NotFound,
+                        "Could not find existing oppfolgingsoppgave with uuid: $uuid"
                     )
-                }
-                ?: call.respond(HttpStatusCode.NotFound, "Could not find existing oppfolgingsoppgave with uuid: $uuid")
+            }
         }
+
         delete("/{$huskelappParam}") {
-            val oppfolgingsoppgaveUuid = UUID.fromString(call.parameters[huskelappParam])
-            val veilederIdent = call.getNAVIdent()
+            call.checkVeilederTilgang(
+                action = API_ACTION,
+                veilederTilgangskontrollClient = veilederTilgangskontrollClient,
+            ) {
+                val oppfolgingsoppgaveUuid = UUID.fromString(call.parameters[huskelappParam])
+                val veilederIdent = call.getNAVIdent()
+
+                val oppfolgingsoppgave = oppfolgingsoppgaveService.getOppfolgingsoppgave(uuid = oppfolgingsoppgaveUuid)
+                if (oppfolgingsoppgave == null) {
+                    call.respond(HttpStatusCode.NotFound)
+                } else {
+                    oppfolgingsoppgaveService.removeOppfolgingsoppgave(
+                        oppfolgingsoppgave = oppfolgingsoppgave,
+                        veilederIdent = veilederIdent
+                    )
+                    call.respond(HttpStatusCode.NoContent)
+                }
+            }
+        }
 
-            val oppfolgingsoppgave = oppfolgingsoppgaveService.getOppfolgingsoppgave(uuid = oppfolgingsoppgaveUuid)
-            if (oppfolgingsoppgave == null) {
-                call.respond(HttpStatusCode.NotFound)
+        post(oppfolgingsoppgaverBulkPath) {
```

Comment:
Skulle vi bare gjort slik? 😊 Kanskje litt lettere å lese path og intensjonen med endepunktet?
```suggestion
        post("/oppfolgingsoppgaver") {
```
Og kanskje navngitt den anderledes?🤔 Slik url'en ser ut nå ville gjettet at den lager en ny ressurs av oppfolgingsopgave. Siden vi allerede er på tynn is sånn REST messig tror jeg det er viktigere å dokumentere at vi fraviker fra konvensjonen her.
Hva med noe sånn?👇
```
post("/get-oppfolgingsoppgaver") {
post("/oppfolgingsoppgaver/get") {
```

Reply:
Det som er digg med variablen er at vi kan bruke den i testene også, uten å definere ny variabel der. 

Men enig i at vi kan endre navn! Likte `/get-oppfolgingsoppgaver` da. https://softwareengineering.stackexchange.com/a/440691

Reply:
Tenker generelt at vi ikke vil tilpasse prodkode til testkode. Heller ikke sånn veldig ofte at routes endres så hadde tenkt at det ikke er så stort tap mtp. DRY. Bare litt lettere å skjønne hensikten til et endepunkt om det står i råtekst. Ikke superviktig, gjør det du tenker er best 😊

Reply:
Ja, fair point det 👍🏼 Jeg endrer